applications/tari_merge_mining_proxy/README.md

```diff
@@ -21,25 +25,17 @@ Follow build instructions for XMRig here:
 Then create a configuration for XMRig using the wizard here:
 `https://xmrig.com/wizard`
 
-When using the wizard you need to use `Add daemon` and the following public stagenet wallet address can be used:
-`55LTR8KniP4LQGJSPtbYDacR7dz8RBFnsfAKMaMuwUNYX6aQbBcovzDPyrQF9KXF9tVU6Xk3K8no1BywnJX6GvZX8yJsXvt`
-
-Take note of the `Host` and `Port` that was specified in the configuration.
+When using the wizard you need to use `Add daemon`. The following public stagenet wallet address can be used (or you can use your own stagenet wallet address):
```

Comment:
What's the significance of this choice?

I think you must explain that this is the address that testnet **Monero** will be paid into. And since we're not really interested in that use the 55LTR address, or if you want the monero then use your own, etc.

Reply:
I will expand on it.

No real significance, just a stagenet wallet address that is the public domain.

Reply:
Have expanded on this